tox.ini

```diff
@@ -21,6 +21,7 @@ deps =
     h: -rrequirements.txt
 passenv =
     TEST_DATABASE_URL
+    PYTEST_ADDOPTS
```

Comment:
For consistency let's pass this through to all pytest environments (so, `functional` too).